RLTSquare/Ccq/Console/Command/HelloWorld.php

```diff
@@ -0,0 +1,127 @@
+<?php
+declare(strict_types=1);
+
+namespace RLTSquare\Ccq\Console\Command;
+
+use Exception;
+use Magento\Framework\MessageQueue\PublisherInterface;
+use Magento\Framework\Serialize\Serializer\Json;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\Console\Command\Command;
+use Symfony\Component\Console\Input\InputArgument;
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use RLTSquare\Ccq\Api\Data\CcqInterface;
+
+/**
+ * @author Afzel Arshad
+ * Cli command for message queue
+ */
+class HelloWorld extends Command
+{
+    /**
+     *  Topic name
+     */
+    const TOPIC_NAME = 'rltsquare.hello.world';
+
+    /**
+     * const name int|string
+     */
+    private const NAME = 'name';
+    /**
+     * const age int|string
+     */
+    private const AGE = 'age';
+    /**
+     * @var PublisherInterface
+     */
+    protected PublisherInterface $publisher;
+    /**
+     * @var LoggerInterface
+     */
+    protected LoggerInterface $logger;
+    /**
+     * @var Json
+     */
+    protected Json $json;
+    /**
+     * @var CcqInterface
+     *
+     */
+    protected CcqInterface $cq;
+
+
+    /**
+     * @param PublisherInterface $publisher
+     * @param LoggerInterface $logger
+     * @param Json $json
+     * @param CcqInterface $cq
+     */
+    public function __construct(
+        PublisherInterface $publisher,
+        LoggerInterface $logger,
+        Json $json,
+        CcqInterface $cq
+    ) {
+        $this->publisher = $publisher;
+        $this->logger = $logger;
+        $this->json = $json;
+        $this->cq = $cq;
+        parent::__construct();
+    }
+
+    /**
+     *  OutputInterfaceInitialization of the command.
+     * @return void
+     */
+    protected function configure(): void
+    {
+        $this->setName('rltsquare:hello:world');
+        $this->setDescription('Commands to take 2 argument a string and num and print it on console');
+        $this->addArgument(
+            self::NAME,
+            null,
+            InputArgument::OPTIONAL,
```

Comment:
Overlooked logic - The Symfony\Component\Console\Input\InputOption would be more suitable for task - see https://symfony.com/doc/current/console/input.html#using-command-arguments